app/views/suppliers/show.html.erb

```diff
@@ -1,3 +1,14 @@
 <div id="supplier_home">
-  <h1><%= @supplier.name %> Supplier Page</h1>
+	<h1><%= @supplier.name %> Supplier Page</h1>
+	<ul>
+		<% @listings.each do |listing|  %>	
+			<li>
+				<% item = Item.find(listing.item_id)  %>
```

Comment:
I feel like this logic should be linked to the item. So Suppliers can call on items.
supplier.items that would return all items for a supplier
but we spoke with another team member who didn't not believe this was the way to handle the relationship. So I would like to see which fiinal implementation makes the team happy before refactoring